sdk/.gitignore

```diff
@@ -33,6 +33,11 @@ target
 /compatibility/head_sdk/
 .bazelrc.local
 .bazelproject
+/build-profile.json
+/test-profile.json
+/build-events.json
+/test-events.json
+/logs
```

Comment:
I've started getting these in my directory, which annoyingly gets included by search programs like ripgrep.